storage_service/locations/models/pipeline.py

```diff
@@ -130,6 +130,8 @@ def create_default_locations(self, shared_path=None):
                     # Fetch existing location
                     location = Location.objects.get(uuid=uuid)
                     assert location.purpose == p['purpose']
+                location.default = True
+                location.save()
```

Comment:
I do this so the new default locations (when AM is installed) are marked as defaults. I haven't found a better way to do this but I may spend some more time.